src/utils/formHelpers.js

```diff
@@ -0,0 +1,16 @@
+import Joi from "joi";
+
+export function getErrorClass(error) {
+  if (!error) return "";
+
+  return "inputError";
+}
+
+export const formSchemaObject = {
```

Comment:
this is called formschemaobject but it seems email specific

Reply:
I will add the required parts of the form validation later when refactor all the forms in the project. I found that email input is used three times the project and each time I create a new validation schema object. When I need to change the email validation object for example, I have to change it in three places as happened yesterday when I added the translation. So I decided to create that object globally and use it in all places.

I rename it to `validationSchemaObject` I think that is more clear than `formschemaobject`.
@louisrli

Reply:
you can't "refactor all the forms in the project" after code freeze. that sounds like a really good way to break everything :)